lib/wallet/send.dart

```diff
@@ -24,6 +29,83 @@ class _SendState extends State<Send> {
 
   @override
   Widget build(BuildContext context) {
+    final formatter = NumberFormat();
+    formatter.minimumFractionDigits = 0;
+    formatter.maximumFractionDigits = 0;
+    final widgets = [
+      const Text("Encoded Invoice", style: TextStyle(color: Colors.grey, fontSize: 18)),
+      Focus(
+        onFocusChange: (hasFocus) async {
+          if (encodedInvoice.isEmpty || hasFocus) {
+            invoice = null;
+            return;
+          }
+
+          try {
+            invoice = await api.decodeInvoice(invoice: encodedInvoice);
+            setState(() {
+              invoice = invoice;
+            });
+          } catch (error) {
+            FLog.error(text: "Failed to decode invoice $encodedInvoice.", exception: error);
+            ScaffoldMessenger.of(context).showSnackBar(SnackBar(
+              backgroundColor: Colors.red,
+              content: Text("Failed to decode invoice. Error: " + error.toString()),
```

Comment:
🔧 I think the user likely cannot make sense of the extra error context or do anything with that extra information. I would omit it and just say that they provided an invalid invoice.

Reply:
different perspective: IMHO at this stage of our product, any user feedback is helpful - if the user reports odd error message, then it's still good for us. And we also get more data doing internal testing, which is annoying, especially on mobile...